src/CodeGenerator/TypeInfo.cs

```diff
@@ -55,6 +55,17 @@ public class TypeInfo
             { "ImPlotFormatter", "IntPtr" },
             { "ImPlotGetter", "IntPtr" },
             { "ImPlotTransform", "IntPtr" },
+            // internals
+            { "char[5]", "byte*"},
+            { "ImGuiDir*", "IntPtr" },
+            //{ "ImGuiStoragePair", "IntPtr" },
+            { "ImGuiDockRequest", "IntPtr" },
+            { "ImGuiDockNodeSettings", "IntPtr" },
+            { "ImGuiTableColumnIdx", "sbyte" },
+            { "ImGuiTableDrawChannelIdx", "byte"},
+            { "ImGuiContextHookCallback", "IntPtr" },
+            { "ImGuiErrorLogCallback", "IntPtr" },
+            //{ "ImGuiSizeCallback", "IntPtr"}
```

Comment:
This was taken from #273 and I'm not totally sure how it all works or if it's correct.

Reply:
I only remember vaguely, but I think I found those by manual "trial and error". Added them here as they were causing erors in the generated internals C# code.

I got to these values by looking at the existing TypeInfos, was pretty straight forward to define the mappings:
- Structs and pointers were mapped to IntPtr
- IDs were mapped to byte/sbyte (might work if both are sbyte, don't remember if the unsigned byte was intentional)
- Char array was mapped to byte pointer, as chars are represented as bytes and arrays need to be pointers

Reply:
I don't know what should be added to these sections as the PR is 2 years old and I don't know what new types need to be defined.